README.md

```diff
@@ -142,7 +142,7 @@ Here are some repos that use cibuildwheel.
 - [pikepdf](https://github.com/pikepdf/pikepdf)
 - [fathon](https://github.com/stfbnc/fathon)
 - [etebase-py](https://github.com/etesync/etebase-py) - Python bindings to a Rust library using `setuptools-rust`, and `sccache` for improved speed, built on Travis CI.
-
+- [xmlstarlet](https://github.com/dimitern/xmlstarlet) - Python CFFI bindings to XMLStarlet XML processing CLI.
```

Comment:
Could you explain in an abridged version why these are interesting here? (You've already done that in this PR description) "XMLStarlet XML processing CLI" isn't interesting, IMO, but "Python CFFI bindings with true MSVC build" or something like that is.

Reply:
Sure, I'll try to rephrase it better. Thanks!